medical_family/models/medical_family.py

```diff
@@ -18,17 +18,25 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-# #############################################################################
+###############################################################################
 
 from openerp import models, fields
 
 
 class MedicalFamily(models.Model):
     _name = 'medical.family'
-    info = fields.Text(string='Extra Information')
-    name = fields.Char(string='Family', required=True)
-    member_ids = fields.One2many('medical.family_member', 'family_id',
-                                 string='Family Members', )
+    info = fields.Text(
+        string='Extra Information'
+    )
+    name = fields.Char(
+        string='Family',
+        required=True
+    )
+    member_ids = fields.One2many(
+        'medical.patient',
+        'family_id',
+        string='Family Members',
+    )
     _sql_constraints = [
-        ('name_uniq', 'UNIQUE(name)', 'Family Code must be unique!'),
+        ('name_uniq', 'UNIQUE(name)', 'Family name must be unique!'),
```

Comment:
Honestly glad you said something on this. I almost removed it then decided maybe the original creators knew more than me. Removing :)